### ISSUE TITLE
CI: enable Travis (or similar) service for testing PRs

### ISSUE THREAD

**Issue by jirikuncar** [davebshow/aiogremlin#11]
At minimum following commands should pass:

- `python setup.py test`
- `sphinx-build -qnNW docs docs/_build/html`

**Comment by davebshow** [comment]
Agree. I may have time to get to this this week. I would also be happy to accept a PR.
